stream_alert/rule_processor/classifier.py

```diff
@@ -16,13 +16,17 @@
 
 import logging
 
-from collections import OrderedDict
+from collections import namedtuple, OrderedDict
 
 from stream_alert.rule_processor.parsers import get_parser
 
 logging.basicConfig()
 LOGGER = logging.getLogger('StreamAlert')
 
+# Set the below to True when we want to support multiple matching on multiple schema
```

Comment:
> multiple matching on multiple schema matching

![image](https://cloud.githubusercontent.com/assets/25284506/25719426/3e44de9e-30be-11e7-94e8-0b67fdebce58.png)

Make this clear